workers/unity/Packages/com.improbable.gdk.mobile/Editor/MobileLauncherWindow.cs

```diff
@@ -0,0 +1,159 @@
+                    launchConfig.RuntimeIpEditorPrefKey,
+                    EditorGUILayout.TextField(RuntimeIpLabel, launchConfig.RuntimeIp));
+                launchConfig.shouldConnectLocally = EditorGUILayout.Toggle(ConnectLocallyLabel, launchConfig.shouldConnectLocally);
+            }
+            
+            DrawHorizontalLine(10, new Color(0.7f, 0.7f, 0.7f, 1));
+            GUILayout.Label(AndroidSectionLabel, EditorStyles.boldLabel);
+
+            if (GUILayout.Button("Launch Android app"))
+            {
+                AndroidLaunchUtils.Launch(launchConfig.shouldConnectLocally);
+            }
+
+#if UNITY_EDITOR_OSX
+
+            DrawHorizontalLine(10, new Color(0.7f, 0.7f, 0.7f, 1));
+
+            GUILayout.Label(iOSSectionLabel, EditorStyles.boldLabel);
+
+            using (new EditorGUI.IndentLevelScope())
+            {
+                EditorPrefs.SetString(
+                    launchConfig.DevelopmentTeamIdEditorPrefKey,
+                    EditorGUILayout.TextField(DevelopmentTeamIdLabel, launchConfig.DevelopmentTeamId));
+                
+                using (new EditorGUI.DisabledScope(false)) // todo check xcode project
+                {
+                    if (GUILayout.Button("Build XCode project"))
+                    {
+                        iOSLaunchUtils.Build(launchConfig.DevelopmentTeamId);
+                    }
+                }
+                
+                DrawHorizontalLine(8, new Color(0.4f, 0.4f, 0.4f, 1));
+
+                using (new GUILayout.HorizontalScope())
+                {
+                    using (new EditorGUI.DisabledScope(simulatorNames.Length == 0))
+                    {
+                        simulatorNameIndex =
+                            EditorGUILayout.Popup("Simulator Model", simulatorNameIndex, simulatorNames);
+                    }
+
+                    var buttonIcon = new GUIContent(EditorGUIUtility.IconContent("Refresh"))
+                    {
+                        tooltip = "Refresh your simulator list."
+                    };
+
+                    if (GUILayout.Button(buttonIcon, EditorStyles.miniButton, GUILayout.ExpandWidth(false)))
+                    {
+                        launchConfig.availableSimulators = iOSLaunchUtils.RetrieveAvailableiOSSimulators();
+                        simulatorNames = launchConfig.availableSimulators.Keys.ToArray();
+                    }
+                }
+
+                using (new EditorGUI.DisabledScope(simulatorNames.Length == 0))
+                {
+                    if (GUILayout.Button("Launch iOS app on Simulator"))
+                    {
+                        var simulatorUID =
+                            launchConfig.availableSimulators[simulatorNames[simulatorNameIndex]];
+                        iOSLaunchUtils.Launch(true, simulatorUID, true);
+                    }
+                }
+                
+                DrawHorizontalLine(8, new Color(0.4f, 0.4f, 0.4f, 1));
```

Comment:
given we only use two colours, should have static Color variables at the top that we re-use